tools/vlada.py

```diff
@@ -0,0 +1,16 @@
+import gym
+from gym_connect_four import RandomPlayer, ConnectFourEnv, Player, SavedPlayer
+import numpy as np
+
+
+class Vlada(Player):
+    """ Clone of RandomPlayer for runner.py illustration purpose """
```

Comment:
If it's a clone, why it also contains implementation? I guess you can just import existing code, instead of duplicating this one, especially when it contains some useless cycles and `if`s in `get_next_action`.